app/shared/modals/reservation-info/ReservationInfoModal.js

```diff
@@ -107,10 +106,7 @@ class ReservationInfoModal extends Component {
                   && !reservation.staffEvent && (
                     <InfoLabel labelStyle={paymentLabel.labelBsStyle} labelText={t(paymentLabel.labelTextId)} />
                 )}
-                {reservation.needManualConfirmation
-                  && reservation.state !== 'cancelled' && (
-                    <InfoLabel labelStyle={stateLabel.labelBsStyle} labelText={t(stateLabel.labelTextId)} />
-                )}
+                {stateLabel && <InfoLabel labelStyle={stateLabel.labelBsStyle} labelText={t(stateLabel.labelTextId)} />}
```

Comment:
Should the conditions `reservation.needManualConfirmation` and `reservation.state !== 'cancelled'` still be checked?